bydbctl/internal/cmd/health_check.go

```diff
@@ -0,0 +1,63 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/apache/skywalking-banyandb/pkg/test/helpers"
+	"github.com/apache/skywalking-banyandb/pkg/version"
+)
+
+func newHealthCheckCmd() *cobra.Command {
+	healthCheckCmd := &cobra.Command{
+		Use:           "health",
+		Version:       version.Build(),
+		Short:         "Health check",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		RunE: func(_ *cobra.Command, _ []string) (err error) {
+			opts := make([]grpc.DialOption, 0, 1)
+			grpcCert := viper.GetString("grpcCert")
+			if grpcCert != "" {
```

Comment:
We need another flag "insecure" to skip server's cert, which is much helpful when this command is used as the health probe in k8s. When a pod is deployed, we might not know the host name of this workload. Ignore the cert might be suitable for this scenario. 

```go
tlsConfig := &tls.Config{
       InsecureSkipVerify: true,
    }
creds := credentials.NewTLS(tlsConfig)
```

Reply:
Furthermore, the HTTP-related requests also need this flag.

```go
client := resty.New()
client.SetTLSClientConfig(&tls.Config{ InsecureSkipVerify: true })
```